test/Integration/vector-contract-to-fma.mlir

```diff
@@ -0,0 +1,49 @@
+// RUN: tpp-opt %s  | tpp-run -e entry --entry-point-result=void -seed 123 -print > %t.1
+// RUN: tpp-opt %s  --vector-contract-to-fma  | tpp-run -e entry --entry-point-result=void -seed 123 -print > %t.2
+// RUN: diff %t.1 %t.2 | FileCheck %s --check-prefix=DIFF --allow-empty
```

Comment:
Don't need to call `FileCheck`, if the diff is not empty it will return non-zero and the test will fail.

Reply:
Done as expected.